src/pretix/static/pretixpresale/js/ui/main.js

```diff
@@ -383,29 +389,22 @@ $(function () {
     })
     $("select[id^=id_name_parts], input[id^=id_name_parts_], #id_email, #id_street, #id_company, #id_zipcode," +
         " #id_city, #id_country, #id_state").change(function () {
-        if (copy_to_first_ticket) {
-            var $first_ticket_form = $(".questions-form").first().find("[data-addonidx=0]");
-            $first_ticket_form.find("[id$=" + this.id.substring(3) + "]").val(this.value);
-            if (this.placeholder) {
-                $first_ticket_form.find("[placeholder='" + CSS.escape(this.placeholder) + "']").val(this.value);
-            }
-            var label = $("label[for=" + this.id +"]").first().contents().filter(function () {
-                return this.nodeType != Node.ELEMENT_NODE || !this.classList.contains("sr-only");
-            }).text().trim();
-            if (label) {
-                // match to placeholder and label
-                $first_ticket_form.find("[placeholder='" + CSS.escape(label) + "']").val(this.value);
-                var v = this.value;
-                $first_ticket_form.find("label").each(function() {
-                    var text = $(this).first().contents().filter(function () {
-                        return this.nodeType != Node.ELEMENT_NODE || !this.classList.contains("sr-only");
-                    }).text().trim();
-                    if (text == label) {
-                        $("#" + this.getAttribute("for")).val(v);
-                    }
-                });
-            }
-        }
+        if (!copy_to_first_ticket) return;
+
+        var source = this;
+        var source_label = get_label_text_for_id(source);
+
+        var $first_ticket_form = $(".questions-form").first().find("[data-addonidx=0]");
+        var $candidates = $first_ticket_form.find(source.tagName + ":not([type='checkbox'], [type='radio'])");
+        var $match = $candidates.filter(function() {
+            return (
+                this.id.endsWith(source.id.substring(3))
+                || this.placeholder === source.placeholder
+                || this.placeholder === source_label
+                || get_label_text_for_id(this.id) === source_label
+            );
+        }).first();
+        $match.val(this.value).trigger("change");
```

Comment:
`trigger("change")` is needed to make the country input work, but basically disables all copying of input after the first one. See line 410. As a quick fix, we could change line 410 from 
`attendee_address_fields.change(function () {`
to
`attendee_address_fields.on("input", function () {` which is not triggered by the change-event here.